readme.md

```diff
@@ -1,17 +1,31 @@
 # Construct
 
+Construct helps you to quickly setup your computer with the tools we have found helpful here at Segment.
 
-Install construct
+Construct will install the following tools:  
 
-  git clone git@github.com:pivotal-cloudops/construct.git
+  atom
+  commandline-tools
+  homebrew
+  homebrew-casks
+  iterm
+  google-chrome
+  recordit
+  slack
+  vagrant
+  virtualbox
 
-Then run it
+
+To install Construct, open your terminal and type:
+
+  `git clone git@github.com:TeresaNesteby/segment-construct.git`  
```

Comment:
This will be a problem on laptops that are absolutely fresh.  Git does not come for free with OSX.  Furthermore, your download link assumes that the user has a github ssh key.